draftquality/feature_lists/enwiki.py

```diff
@@ -1,6 +1,48 @@
 from revscoring.features import wikitext
 from revscoring.features.modifiers import max, sub
 from revscoring.languages import english
+from revscoring import Feature
+from revscoring.datasources import Datasource
+from nltk.corpus import sentiwordnet as swn
+
+def get_polarity_score(non_stop_tokens):
+    """
+    Gets the positive and negative polarity of the document using SentiWordnet
+    takes the most common sense of the word for efficiency
+    """
+    pos, neg = 0.0, 0.0
+    for t in non_stop_tokens:
+        synsets = list(swn.senti_synsets(t))
+        if synsets:
+            pos += synsets[0].pos_score()
+            neg += synsets[0].neg_score()
+    return [pos, neg]
+
+sentiment_score = Datasource("polarity_score",
```

Comment:
This is going to work OK, but it's good practice to have long form names.  E.g. `"english.sentiment.revision.polarity_score"`